rest/model/commit_queue_test.go

```diff
@@ -162,4 +162,12 @@ Singlewordthatis72characterslonganditshouldntbebrokenupbythewrappingfunc`
 	assert.Equal(utility.ToStringPtr("module1"), data.Modules[0].Module)
 	assert.Equal(utility.ToStringPtr("1234"), data.Modules[0].Issue)
 	assert.Equal("Singlewordthatis72characterslonganditshouldntbebrokenupbythewrappingfunc", data.MessageOverride)
```

Comment:
I'm a bit confused about why this passes if it's only 72 characters

Reply:
It's behaving as expected since we're not breaking up words. In the extreme case of a message having a word of 72+ characters that word will take up a whole line no matter how long it is. This isn't a realistic case anyway